posthog/api/test/test_prompt.py

```diff
@@ -0,0 +1,75 @@
+from typing import Dict, List
+
+from freezegun.api import freeze_time
+from rest_framework import status
+
+from posthog.models import User
+from posthog.models.person.person import Person
+from posthog.models.prompt import PromptSequenceState, experiment_config
+from posthog.test.base import APIBaseTest
+
+
+class TestPrompt(APIBaseTest):
```

Comment:
we should have a cross-team and cross user test similar to these https://github.com/PostHog/posthog/blob/master/posthog/api/test/test_insight.py#L1243

i.e. 

* user A can't view/amend prompts for user B
* user in Team A can't view/amend prompts for user in Team B

Reply:
Brilliant, caught a bug thanks to this, added a test, thanks